elytron/ELY-1778-enhanced-audit-logging-feature-parity-with-legacy.adoc

```diff
@@ -0,0 +1,124 @@
+= ELY- Enhanced Audit Logging - Feature Parity with Legacy Audit Logging
+:author:            Justin Michael Cook
+:email:             jucook@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Currently, Audit Logging in Elytron does not have the same capabilities as Legacy Security. This task is to enhance Elytron Audit Logging to have the same capabilities as Legacy Security Audit Logging through the addition of an @Audit annotation and additional audit events.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/ELY-1778[ELY-1778]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1230[EAP7-1230]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+
+[X] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* Security
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* There should be the ability to specify a Java class should be logged by using an @Audit Java annotation, similarly to what was https://developer.jboss.org/wiki/PicketBoxSecurityAnnotations#jive_content_id_SecurityAudit_Annotation[available in Picketbox]
+** This annotation would be available to be added onto a class for signifying when that class is called it should be logged
```

Comment:
"Calling a class" mean any method from it or making an instance of it?

Reply:
Just clarified this in the document - it would be calling methods from it.